### PR TITLE
Feature/profile page

### PR DESCRIPTION
**Pull request by LCD344**

### CODE REVIEW

File: app/Http/Controllers/UserController.php [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+        $user = $request->commit();
+        $message = __('profile/update.updatedLanguage');
+        return back()->with('alert', $message);
+    }
+
+    public function destroy()
+    {
+        //
+        $user = Auth::user();
```
Comment (LCD344) [inline app/Http/Controllers/UserController.php]: chane the Auth::user(); to $request->user();

File: package.json [hunk 2]
```diff
@@ -27,5 +27,11 @@
     "vue-template-compiler": "^2.2.6",
     "vuejs-paginate": "^0.8.4",
     "vuetable-2": "^1.3.1"
+  },
+  "dependencies": {
```
Comment (LCD344) [inline package.json]: these dependencies should probably be deleted

File: resources/lang/es/auth.php [hunk 3]
```diff
@@ -21,9 +21,12 @@
     'language' => 'Idioma',
     'login' => 'Ingresar',
     'register' => 'Registrate',
-    'notMemberYet' => '¿Todavía no tienes una cuenta?',
+    'notMemberYet' => 'Todavía no tenés una cuenta',
```
Comment (LCD344) [inline resources/lang/es/auth.php]: why was this changed? tenes is a form of spanish only used in argentina. and its stupid

File: routes/web/user.php [hunk 4]
```diff
@@ -0,0 +1,11 @@
+<?php
+
+Route::group(['prefix' => 'user'],function(){
+	Route::get('/', 'UserController@edit');
+	Route::delete('/', 'UserController@destroy');
+	Route::get('/matches', 'UserController@getMatches');
+	Route::patch('/user/username','UserController@updateUsername');
```
Comment (LCD344) [inline routes/web/user.php]: these urls are a bit weird. you can probably drop the extra user from them

**Comment by Mahmoud1Ghonim** [comment]
Fixed
